### ISSUE TITLE
Training Data size

### ISSUE THREAD

**Issue by selvamuthu** [mercerworkforceanalyticsSG/Hiring-Challenge#2]
Hello,

I found out that the data is quite biased. I mean we only have limited data (500-600 records) with respect to people who left the company whereas we have more than 7000 active employee records. I feel this might lead to a drop in accuracy while prediction. Am I right?

**Comment by asothmann** [comment]
Yes that observation is valid. More often that not with HR data we will be faced with imbalanced classes. When looking at classification accuracy, we tend to see the models overfitting to the 'largest' class. My advice would be to consider the performance measure you use and try a number of different algorithms. More importantly, think about the implications of your dataset when interpreting the output and making recommendations. 

Aaron
